### DISCUSSION TITLE
Rename wheel model for modern trucks

### DISCUSSION THREAD

**Discussion by YohDeadfall** [voxeltycoon/issues#439] [Modding]
There is no European rims in the game, but there is `truck_wheel_3` which represents a spider wheel for modern trucks. It leads to adding custom wheels which each modern model. To fix this the old model could be renamed to `truck_wheel__spider_3` to avoid confusion, and new one could be added later as `truck_wheel_3`.

**Comment by YohDeadfall** [comment]
Spent some time and redesigned existing wheels, so now they looks better. As a bonus added a super single spider wheel. As the next step I will provide the same but with European rims.

Take a look [here](https://github.com/YohDeadfall/voxeltycoon).

**Comment by YohDeadfall** [comment]
New wheels just added and as before in three variations: steer, drive and wide.

**Comment by victorleblais** [comment]
Is it really a big deal though? The file name has no impact on what is visible in-game.

**Reply by YohDeadfall** [reply]
Not so much, but this makes it easier to navigate in files without looking their content. Anyway, the game stepped into early access, and it might be a breaking change.
